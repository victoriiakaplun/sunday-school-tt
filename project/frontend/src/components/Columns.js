import React from 'react';

function Columns({ children }) {
  return <div className="columns is-centered">{children}</div>;
}

export default Columns;
