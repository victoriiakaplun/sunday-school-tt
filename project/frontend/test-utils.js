import React from 'react'; // eslint-disable-line no-unused-vars

import { initFontAwesomeLibrary } from './src/utils';

initFontAwesomeLibrary();

// import '@testing-library/jest-dom/extend-expect';

// re-export everything
export * from '@testing-library/react';
