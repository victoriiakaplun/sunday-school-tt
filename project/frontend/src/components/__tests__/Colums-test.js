import { render } from 'test-utils';
import Columns from '../Columns';

describe('Columns', () => {
  it('renders correctly', () => {
    const { container } = render(<Columns>Columns entry</Columns>);
    expect(container.firstChild).toMatchSnapshot(`
      <div 
      class="columns is-centered">
      Column entry
      </div>
    `);
  });
});
