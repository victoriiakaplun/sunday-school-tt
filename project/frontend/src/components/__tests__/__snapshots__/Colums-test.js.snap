// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Columns renders correctly: 
      <div 
      class="columns is-centered">
      Column entry
      </div>
     1`] = `
<div
  class="columns is-centered"
>
  Columns entry
</div>
`;
